sitzverteilungvue/apigateway/pom.xml

```
@@ -5,13 +5,6 @@
     <modelVersion>4.0.0</modelVersion>
 
 
-    <parent>
-        <groupId>de.muenchen</groupId>
-        <artifactId>muenchen-pom</artifactId>
-        <version>2.1.0</version>
-    </parent>
-
-
     <groupId>de.muenchen.sitzverteilungvue.apigateway</groupId>
```

```suggestion
    <groupId>de.muenchen.wahl.sitzverteilung.frontend</groupId>
```
Siehe oben. Zu hoch in der Hierarchie, besser unter de.muenchen.wahl und die groupID soll m.E in den 2 Gruppen die wir haben wollen geteilt, backend oder service und frontend